ksqldb-rest-app/src/test/java/io/confluent/ksql/api/integration/RestoreCommandTopicIntegrationTest.java

```diff
@@ -173,4 +256,27 @@ private boolean isDegradedState() {
   private List<KsqlEntity> makeKsqlRequest(final String sql) {
     return RestIntegrationTestUtil.makeKsqlRequest(REST_APP, sql);
   }
+
+  public static void writeToBackupFile(
+      final CommandId commandId,
+      final Command command,
+      final Path backUpFileLocation
+  ) throws IOException {
+    FileOutputStream writer;
+    try {
+      writer = new FileOutputStream(new File(String.valueOf(backUpFileLocation)), true);
+    } catch (final FileNotFoundException e) {
+      throw new KsqlException(
+          String.format("Failed to open backup file: %s", backUpFileLocation), e);
+    }
+
+    final byte[] keyValueSeparator = ":".getBytes(StandardCharsets.UTF_8);
+    final byte[] newLine = "/n".getBytes(StandardCharsets.UTF_8);
+
+    writer.write(InternalTopicSerdes.serializer().serialize("", commandId));
+    writer.write(keyValueSeparator);
+    writer.write(InternalTopicSerdes.serializer().serialize("", command));
+    writer.write(newLine);
+    writer.flush();
```

Comment:
This looks like what the `BackupReplayFile` does. Can it be reused?
```
 // Just create another write() that accept the key and value instead of a consumer
BackupReplayFile.writable(backupFileLocation.toFile())
    .write(commandIdInBytes, commandInBytes);
```

Reply:
I reused it now